blog/_posts/case-study-technical-prism-sideshift.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Technical Case Study: How to enable Bitcoin payments for merchants at events using BTCPay Server"
+date: 2023-10-03
+author: RockstarDev, MrKukks
+category:
+  - "stories"
+tags:
+  - "case-study"
+  - "point of sale"
+  - "research"
+  - "merchants"
+  - "sideshift"
+  - "prism"
+coverImage: "/images/case-study-technical-prism-sideshift/technical-case-study-cover.png"
+---
+
+Recently, we published [a case study highlighting the success of the Baltic Honey Badger 2023](https://blog.btcpayserver.org/case-study-hodlhodl-2023/) - a Bitcoin conference where all merchants exclusively used Bitcoin to facilitate payments for food and drinks.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/mDeuHh9gqQM?si=I8pO_Ed5lQUm8m65" title="Case Study: How Hodl Hodl used BTCPay to enable 20 merchants at a conference to accept bitcoin" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
We don't embed videos like this, especially since for some reason it doesn't look good on mobile.

```suggestion
[![Case Study Riga](https://img.youtube.com/vi/mDeuHh9gqQM/mqdefault.jpg)](https://www.youtube.com/watch?v= mDeuHh9gqQM)
```